### PR TITLE
Issue #899 Minor documentation improvements

### PR DESCRIPTION
**Pull request by ljacomet**

### CODE REVIEW

File: docs/src/docs/asciidoc/user/getting-started.adoc [hunk 1]
```diff
@@ -67,6 +65,8 @@ NOTE: See link:usermanaged{outfilesuffix}[the user managed cache documentation]
 
 === Tiering
 
+Ehcache 3, as in previous version, offers a tiering model to allow to store growing amount of data on slower tiers.
```
Comment (chrisdennis) [inline docs/src/docs/asciidoc/user/getting-started.adoc]: Ehcache 3, as in previous versions, offers a tiering model to allow storing increasing amounts of data on slower tiers.

File: docs/src/docs/asciidoc/user/getting-started.adoc [hunk 2]
```diff
@@ -76,7 +76,10 @@ include::{sourcedir}/impl/src/test/java/org/ehcache/docs/GettingStarted.java[tag
 
 <1> If you wish to use off-heap, you'll have to define resource pool, giving the memory size you want to allocate.
 
-Do not forget to define in the java options the -XX:MaxDirectMemorySize option, according to the off-heap size you intend to use.
+The example above allocates a very small amount of off-heap.
+Remember that data stored off-heap will have to be serialized and deserialized.
+You should thus favor off-heap for large amounts of data where storing them on heap would have too severe impacts on garbage collection.
```
Comment (chrisdennis) [inline docs/src/docs/asciidoc/user/getting-started.adoc]: You should thus favor off-heap for large amounts of data where on heap would have too severe an impact on garbage collection.

File: docs/src/docs/asciidoc/user/getting-started.adoc [hunk 3]
```diff
@@ -76,7 +76,10 @@ include::{sourcedir}/impl/src/test/java/org/ehcache/docs/GettingStarted.java[tag
...
+Do not forget to define in the java options the `-XX:MaxDirectMemorySize` option, according to the off-heap size you intend to use.
```
Comment (chrisdennis) [inline docs/src/docs/asciidoc/user/getting-started.adoc]: This seems to imply setting these to the same value... thats not really what we want... but I'm not sure what we should say instead.

File: docs/src/docs/asciidoc/user/getting-started.adoc [hunk 4]
```diff
@@ -207,6 +216,9 @@ In this particular example, the `OddKeysEvictionVeto` class will veto eviction o
 By the time the cache manager gets closed, only mappings with odd keys should be left in the cache as there isn't
 enough room for mapping with even keys.
 
+NOTE: Veto status is computed when a mapping is written to the cache.
+This means that proper veto implementation are expected to be constant for a key-value pair.
```
Comment (chrisdennis) [inline docs/src/docs/asciidoc/user/getting-started.adoc]: implementations

File: docs/src/docs/asciidoc/user/getting-started.adoc (hunk 3 above)
Comment (ljacomet) [inline docs/src/docs/asciidoc/user/getting-started.adoc]: Let's not overthinking it right now, we can work on something better when we do a full section on tiering configuration, pros and cons, etc...

Comment (chrisdennis) [inline docs/src/docs/asciidoc/user/getting-started.adoc]: ok

**Pull request merged**